packages/esm/core/src/Engines/WebGL/engine.adapterHelpers.ts

```diff
@@ -0,0 +1,157 @@
+import { createCubeTexture } from "./Extensions/cubeTexture/cubeTexture.webgl.js";
+import { createDynamicTexture, updateDynamicTexture } from "./Extensions/dynamicTexture/dynamicTexture.webgl.js";
+import { createRawTexture, createRawTexture3D, createRawTexture2DArray, createRawCubeTexture } from "./Extensions/rawTexture/engine.rawTexture.webgl.js";
+import type { IWebGLEnginePublic } from "./engine.webgl.js";
+import {
+    createTexture,
+    _createHardwareTexture,
+    _releaseTexture,
+    _bindTextureDirectly,
+    _unpackFlipY,
+    _bindTexture,
+    _executeWhenRenderingStateIsCompiled,
+    _getShaderProcessor,
+    _preparePipelineContext,
+    bindSamplers,
+    bindUniformBlock,
+    createPipelineContext,
+    setDepthStencilTexture,
+    setTexture,
+    setTextureArray,
+    setTextureFromPostProcess,
+} from "./engine.webgl.js";
+import { getCaps, getHostDocument, getLoadedTexturesCache, getRenderingCanvas } from "../engine.base.js";
+import type { InternalTexture } from "@babylonjs/core/Materials/Textures/internalTexture.js";
+import { InternalTextureSource } from "@babylonjs/core/Materials/Textures/internalTexture.js";
+import type { Scene } from "@babylonjs/core/scene.js";
+import type { Nullable } from "@babylonjs/core/types.js";
+import { _createPrefilteredCubeTexture } from "@babylonjs/core/Misc/dds.js";
+import { augmentEngineState } from "../engine.adapters.js";
+import { _loadFile } from "../engine.tools.js";
+import { bindUniformBufferBase } from "./Extensions/uniformBuffer/uniformBuffer.webgl.js";
+
+export const createPrefilteredCubeTextureWebGLAdapter = {
+    getCaps,
+    _bindTextureDirectly,
+    _unpackFlipY,
+    createCubeTexture,
+};
+
+export const effectWebGLAdapter = {
+    getHostDocument,
+    _getShaderProcessingContext: () => null,
+    _getShaderProcessor,
+    _loadFile,
+    createPipelineContext,
+    _preparePipelineContext,
+    _executeWhenRenderingStateIsCompiled,
+    bindSamplers,
+    _bindTexture,
+    setTexture,
+    setDepthStencilTexture,
+    setTextureArray,
+    setTextureFromPostProcess,
+    bindUniformBufferBase,
+    bindUniformBlock,
+};
+
+
+export const effectWebGPUAdapter = {

```

Comment:
I think you forgot to remove this one from the file?

Reply:
yes! thanks. Was forgotten when i restructured the directories. thanks :-)